### DISCUSSION TITLE
General questions

### DISCUSSION THREAD

**Discussion by orbeckst** [Becksteinlab/MDPOW#207] [Announcements]
We encourage everyone to ask general questions in this Discussion forum (under “[General](https://github.com/Becksteinlab/MDPOW/discussions/categories/general)”). If it's a very specific question you can also post it under [Q&A](https://github.com/Becksteinlab/MDPOW/discussions/categories/q-a).

If you find a bug (which includes unclear or incorrect documentation or examples), please open an issue in the [issue tracker](https://github.com/Becksteinlab/MDPOW/issues). 

If you have a question or comment on something for which there’s already an issue open then please add your comments there. Having all relevant information in one place makes it a lot easier to solve problems. 

----

Apologies in advance if answers take some time. Nobody is working full time on this project and it is provided in the hope that it may be useful to others. But we don't really have the capacity to provide meaningful support. Everybody is very much encouraged to help each other. — @orbeckst

**Comment by a-ws-m** [comment]
Hi, I've been trying to use MDPOW to compute partition coefficients with the CHARMM forcefield. The first thing I've noticed is that the `system.top` file that is copied by `mdpow-equilibrium` (from [here](https://github.com/Becksteinlab/MDPOW/blob/develop/mdpow/templates/system.top)) has the `oplsaa.ff` directory and files hardcoded. Is this intended behaviour?

**Reply by orbeckst** [reply]
Hi @a-ws-m, could you please post your question as a separate, new question in the [Q&A section](https://github.com/Becksteinlab/MDPOW/discussions/categories/q-a)? Thanks.

**Reply by a-ws-m** [reply]
Sure, I've copied it [here](https://github.com/Becksteinlab/MDPOW/discussions/265#discussion-5534666).
